src/Windows/Avalonia.Win32/PopupImpl.cs

```diff
@@ -17,6 +17,14 @@ class PopupImpl : WindowImpl, IPopupImpl
         [ThreadStatic]
         private static IntPtr s_parentHandle;
 
+        public void DisableCloseButton()
```

Comment:
These can be moved from `PopupImpl` if the feature is moved from `WindowBase` to `Window`.